src/components/UserLabel/types.ts

```diff
@@ -0,0 +1,23 @@
+import type React from 'react';
+
+import type {DistributiveOmit} from '../../types/utils';
+import type {AvatarProps} from '../Avatar';
+import type {DOMProps, QAProps} from '../types';
+
+export type UserLabelType = 'person' | 'email' | 'empty';
+export type UserLabelText = {value: string; content: React.ReactNode} | string;
+export type UserLabelSize = 's' | 'm';
+export type UserLabelView = 'outlined' | 'clear';
+
+export interface UserLabelProps extends DOMProps, QAProps {
+    type?: UserLabelType;
+    avatar?:
+        | DistributiveOmit<AvatarProps, 'size' | 'className'>
+        | string
+        | React.ReactElement<unknown>;
+    text: UserLabelText;
+    size?: UserLabelSize;
+    view?: UserLabelView;
+    onClick?: (value: string) => void;
+    onClose?: (value: string) => void;
```

Comment:
DOM handlers should have native event, like other components do. `value` looks redundant in this component

Reply:
`value` can be useful for cases when we render components from lists